### ISSUE TITLE
Bluetooth 

### ISSUE THREAD

**Issue by juerse** [syssi/esphome-seplos-bms#65]
Hi
Is it possible to use bluetooth integration?

**Comment by syssi** [comment]
Not yet.

**Comment by juerse** [comment]
when do you think it will be possible?
Thank you

**Comment by syssi** [comment]
I don't provide estimates because it makes people angry if I miss the deadline. ;-) The implementation will take time.

**Comment by juerse** [comment]
Ok thank you

**Comment by andreas-bulling** [comment]
That'd be fantastic, although the wired connection also works really fine.

Even better would be if it was possible to change settings through ESPHome without having to use the (desktop) app.

**Comment by saviothecnic** [comment]
When it will also be available in beta
I'll be happy to try

**Comment by Dukat-Gul** [comment]
Hi @syssi just a quick status check. Is the ble / add bluetooth support branch considered a work in progress or is it something you are interrsted having others in the community perform testing?

**Comment by syssi** [comment]
This is correct. At the moment there is no testing needed yet because the implementation isn't ready.

**Comment by syssi** [comment]
The implementation is ready and merged into `main`. Please give it a try! :-)

**Comment by saviothecnic** [comment]
WOW 
Great news!
Question for the Seplos V 2.0 BMS
Which example should I use?
I have 4 BMSs. Do I need to use 4 ESP32s, or can I use just one for all 4?

**Comment by syssi** [comment]
Please start with a single BMS. :-) If the ESP is powerful there should be multiple BLE connections possible.
